central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -130,24 +156,21 @@ func (ds *datastoreImpl) AddProcessListeningOnPort(
 	// timestamp
 	// * If no existing PLOP is present, they will create a new closed PLOP
 	for _, val := range completedInBatch {
-		indicatorID := ""
 		var processInfo *storage.ProcessIndicatorUniqueKey
 
-		key := getPlopProcessUniqueKey(val)
-
-		if indicator, ok := indicatorsMap[key]; ok {
-			indicatorID = indicator.GetId()
-			log.Debugf("Got indicator %s: %+v", indicatorID, indicator)
-		} else {
-			countMetrics.IncrementOrphanedPLOPCounter(val.GetClusterId())
-			log.Warnf("Found no matching indicators for %s", key)
-			processInfo = val.Process
-		}
-
+		indicatorID := getIndicatorIDForPlop(val)
```

Comment:
same comment above as this returning empty and do we need to handle that case or not.